backend/yarn.lock

```diff
@@ -0,0 +1,1511 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+abab@^1.0.3:
```

Comment:
this file probably shouldn't be in `backend`

Reply:
after mentioning it I think it's better to keep the files and not ignore them.
https://yarnpkg.com/en/docs/migrating-from-npm 
I think we should keep yarn.lock file in both files

> Yarn will generate a yarn.lock file within the root directory of your package. You don’t need to read or understand this file - just check it into source control. When other people start using Yarn instead of npm, the yarn.lock file will ensure that they get precisely the same dependencies as you have.

Reply:
:+1: